desc/objectives/normalization.py

```diff
@@ -26,4 +26,9 @@ def compute_scaling_factors(eq):
     scales["F"] = scales["p"] / scales["a"]
     scales["f"] = scales["F"] * scales["V"]
     scales["Psi"] = abs(eq.Psi)
+    # replace 0 scales to avoid normalizing by zero
```

Comment:
this is good, but also for the particular case of `iota=0` I think the thing you had for `B_P` also makes sense, so that at least `I` will be order 1/mu0 instead of just 1